Source/Bifrost.FluentValidation/MetaData/ValidationMetaDataGenerator.cs

```diff
@@ -72,19 +79,21 @@ void GenerateForValidator(IValidator inputValidator, TypeMetaData metaData, stri
 
             foreach (var member in members)
             {
-                var rules = descriptor.GetRulesForMember(member.Key);
-                foreach (var rule in rules)
+                var isModelRule = member.Key == ModelRule<string>.ModelRulePropertyName || member.Key == string.Empty;
```

Comment:
member.Key now returns string.Empty for model rules after update. (Because of our custom PropertyNameResolver.) This was changed back in FluentValidation 4.